plugins/postgres/dbt/include/postgres/macros/adapters.sql

```diff
@@ -14,6 +14,25 @@
   );
 {%- endmacro %}
 
+{% macro postgres__get_create_index_sql(relation, index_dict) -%}
+  {%- set comma_separated_columns = ", ".join(index_dict['columns']) -%}
+
+  {#- We append the current timestamp to the index name because otherwise the index will -#}
+  {#- only be created on every other run. See -#}
+  {#- https://github.com/fishtown-analytics/dbt/issues/1945#issuecomment-576714925 for -#}
+  {#- an explanation. -#}
+  {%- set index_name = md5("_".join([run_started_at.strftime("%s"), relation.identifier] + index_dict['columns'])) -%}
```

Comment:
I'm not sure if this `md5` call will work since I don't understand how the dbt core utils are included and made available in `.sql` files.